CoinDock-app/src/App/Routes/AuthRoutes.js

```diff
@@ -0,0 +1,73 @@
+import { useIsAuthenticated } from "App/Auth/hooks";
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import Account from "Screens/Account/Account";
+import Dashboard from "Screens/Dashboard/Dashboard";
+import RecoveryCodeTestStep from "Screens/SignUp/RecoveryCodeTestStep";
+import RecoveryCodeBoxStep from "Screens/SignUp/RecoveryStep/RecoveryStep";
+import RecoveryCodeBoxStepAccount from "Screens/SignUp/RecoveryStep/RecoveryStepAccount";
+import RecoveryCodeTestStepAccount from "Screens/SignUp/RecoveryCodeTestStep/RecoveryCodeTestStepAccount";
+
+import AccountSettings from "Screens/Account/settings/AccountSettings";
+import ProfileSettings from "Screens/Account/settings/ProfileSettings";
+import SystemSettings from "Screens/Account/settings/SystemSettings";
+import ProfileName from "Screens/Account/Profile/ProfileName";
+import DateofBirth from "Screens/Account/Profile/DateOfBirth";
+import Country from "Screens/Account/Profile/Country";
+import Accountemail from "Screens/Account/AccountS.js/Accountemail";
+import Accountpassword from "Screens/Account/AccountS.js/Accountpassword";
+import Primary from "Screens/Account/System.js/Primary";
+import Secondary from "Screens/Account/System.js/Secondary";
+const AuthRoutes = (props) => {
+  const isAuthenticated = useIsAuthenticated();
+
+  if (!isAuthenticated) {
+    return <React.Fragment />;
+  }
+  // if (signupsteps?.data?.results?.step_details?.step_1_completed) {
```

Comment:
Can you remove the commented part please if not needed

Reply:
Removed